Makefile

```diff
@@ -153,6 +153,9 @@ test-vrg-vr: generate manifests setup-envtest
 test-vrg-vs: generate manifests setup-envtest
 	go test ./controllers -coverprofile cover.out $(GO_TEST_GINKGO_ARGS) -ginkgo.focus VolumeReplicationGroupVolSync
 
+test-vrg-kubeobjects: generate manifests setup-envtest
+	go test ./controllers -coverprofile cover.out $(GO_TEST_GINKGO_ARGS) -ginkgo.focus VRG_KubeObjectProtection
+
```

Comment:
I don't see the ginkgo tests in this PR, so I presume this is coming in a subsequent one.

Reply:
Added this in forced update on 385ec610d9e486f5bf28a609513052c5bf26d63d